sdk/test-utils/recorder-v2/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../../tsconfig.package",
+  "compilerOptions": {
+    "declarationDir": "./types",
+    "target": "es5",
+    "outDir": "./dist-esm",
+    "lib": [
+      "dom",
+      "es5",
+      "es6",
+      "es7",
+      "esnext"
+    ]
+  },
+  "exclude": [
```

Comment:
You don't have to specify these because they aren't part of `"include"`. If you specify `"include"`, then nothing outside of the includes will be included in the first place.